docs/monitoring.md

```diff
@@ -269,17 +303,24 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Details for the storage status of a given RDD</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/logs</code></td>
-    <td>Download the event logs for all attempts of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/logs</code></td>
+    <td>Download the event logs for all attempts of the given application as files within
+    a zip file.
+    </td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/[attempt-id]/logs</code></td>
+    <td>Download the event logs for a specific application as a zip file.</td>
   </tr>
 </table>
 
-When running on Yarn, each application has multiple attempts, so `[app-id]` is actually
-`[app-id]/[attempt-id]` in all cases.
+The number of jobs and stages which can retrieved is constrained by the same garbage
```

Comment:
true, it's a sort of meta-level GC: go over the limit and the UI just throws away things. How about:  "the retention policy of the standalone spark UI"